tests/routers/test_notifiers_security.py

```
@@ -0,0 +1,101 @@
+
+import pytest
+from fastapi.testclient import TestClient
+from sqlalchemy.orm import Session
+from app.models import User, Notifier, NotifierType
+from app.security import hash_password
+
+@pytest.fixture
+def authenticated_client(client: TestClient, db_session: Session):
+    """Fixture to get an authenticated client."""
+    username = "authtestuser"
+    password = "password"
```

Ruff (0.15.0)

[error] Possible hardcoded password assigned to: "password"

(S105)